pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -414,9 +414,14 @@ def _check_device_config_and_set_final_flags(
         self._devices_flag = devices
 
         if self._devices_flag in ([], 0, "0"):
+            accelerator_name = (
```

Comment:
`self.accelerator` is not set till this point?

Reply:
It's not